pkgs/misc/universal-android-debloater/default.nix

```diff
@@ -0,0 +1,59 @@
+{ pkgs ? import <nixpkgs> { system = builtins.currentSystem; }
+, stdenv
+, lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "universal-android-debloater";
+  version = "unstable-2023-04-10";
+
+  src = fetchFromGitHub {
+    owner = "0x192";
+    repo = pname;
+    rev = "11f27c671cba278d71296cdef4c5a5dba06add5e";
+    hash = "sha256-5vGn7FRkko44rkMID23XK1YTjFBuhA4Hz8h49gB/L2Q=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
+    allowBuiltinFetchGit = true;
+  };
+
+  nativeBuildInputs = with pkgs; [
+    cargo
+    clang
+    cmake
+    mold
+    pkg-config
+    rustc
+    makeWrapper
+  ];
+
+  buildInputs = with pkgs; [
+    fontconfig
+    freetype
+    libglvnd
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXrandr
+  ];
+
+  doCheck = false;
+
+  buildFeatures = [ "iced/glow" ];
+  postInstall = with pkgs; ''
+    wrapProgram $out/bin/uad_gui \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ fontconfig freetype libglvnd xorg.libX11 xorg.libXcursor xorg.libXrandr xorg.libXi ]} \
+      --suffix PATH : ${lib.makeBinPath [ android-tools ]}
+    '';
+
+  meta = with lib; {
```

Comment:
You don't have to make a separate PR for adding yourself to maintainers-list.nix. Just add an additional commit in this PR. That way, you can mark yourself as the maintainer of this package, by adding `meta.maintainers`.

Reply:
Thank you. I just added the commit